### PR TITLE
Update SettingUpTypo3ManuallyLinux.rst

### PR DESCRIPTION
**Pull request by uwe188**
Add hint for apache configuration to allow .htaccess rewrite rules provide by typo3 installation

### CODE REVIEW

**Comment by linawolf** [comment]
I believe it is out of context to describe how to configure an apache server. I would suggest we link to a good external source instead? @uwe188 do you know a good one?

**Comment by uwe188** [comment]
changed and added link to apache docu

File: Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst [hunk 1]
```diff
@@ -37,8 +37,11 @@ Apache Webserver
    sudo apt-get install apache2
    sudo a2enmod deflate rewrite headers mime expires
 
-
-
+.. note::
+   to allow override with .htaccess files for your typo3 base directory
```
Comment (linawolf) [inline Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:41]:
```suggestion
   To allow override with :file:`.htaccess` files for your  TYPO3 base directory 
```

File: Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst [hunk 2]
```diff
@@ -37,8 +37,11 @@ Apache Webserver
    sudo apt-get install apache2
    sudo a2enmod deflate rewrite headers mime expires
 
-
-
+.. note::
+   to allow override with .htaccess files for your typo3 base directory 
+   change AllowOverride from None to ALL
```
Comment (linawolf) [inline Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:42]:
```suggestion
   change `AllowOverride` from `None` to `ALL`
```

Comment (uwe188) [reply Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:42]: Nice :-) ..Danke

File: Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst [hunk 3]
```diff
@@ -37,8 +37,11 @@ Apache Webserver
    sudo apt-get install apache2
    sudo a2enmod deflate rewrite headers mime expires
 
-
-
+.. note::
+   To allow override with :file:`.htaccess` files for your  TYPO3 base directory
```
Comment (linawolf) [inline Documentation/Appendix/Linux/SettingUpTypo3ManuallyLinux.rst:41]:
```suggestion
   To allow override with :file:`.htaccess` files for your TYPO3 base directory 
```

**Review by linawolf — APPROVED** [review]

**Comment by DavidBruchmann** [comment]
@linawolf please merge

**Review by linawolf — APPROVED** [review]

**Pull request merged**